api/src/controllers/team.js

```diff
@@ -183,6 +186,26 @@ function updateTeamMember(req, res, next) {
     .catch((err) => next(fillErrorObject(500, 'Server error', [err])));
 }
 
+/**
+ * Reset the team member from the database on /team/:teamId/member.
+ * @param {*} req request object, containing team id in the url
+ * @param {*} res response object, the updated team member document
+ * @returns 200: the team member was updated
+ * @returns 404: team is not found
+ * @returns 400: team id is not in a valid hexadecimal format
+ */
+async function resetTeamMember(req, res, next) {
```

Comment:
Curious, why an endpoint to specifically remove team members ? Shouldn’t team member removal be part of the broader delete as well?